dask/dataframe/core.py

```diff
@@ -2822,6 +2824,9 @@ def value_counts(self, split_every=None, split_out=1):
             split_every=split_every,
             split_out=split_out,
             split_out_setup=split_out_on_index,
+            sort=sort,
```

Comment:
The `sort` keyword will probably not provide correct results if there are multiple output partitions.  Each output partition will be sorted but they won't be sorted between each other.  

The number of output partitions is controlled by the `split_out=` parameter.  I recommend that we raise a `NotImplementedError` if `sort and split_out != 1`.

Reply:
ah ok. I hadn't thought through that scenario since my toy example wasn't going down that path. I wasn't sure whether the `sort` kwarg should be added at all in this PR, but it seems to cut the computation time in half (for my toy example), so I thought it might be useful.

Reply:
but isn't that case already handled here: https://github.com/dask/dask/blob/cc952ac401937df54063122d8a05bbacdd024786/dask/dataframe/core.py#L4847-L4854

Reply:
Ah indeed.  Great!